includes/admin/views/html-admin-page-addons.php

```diff
@@ -11,124 +11,148 @@
-						href="<?php echo esc_url( admin_url( 'admin.php?page=wc-addons&section=' . esc_attr( $section->slug ) ) ); ?>">
-						<?php echo esc_html( $section->label ); ?>
-					</a>
-				</li>
-			<?php endforeach; ?>
-		</ul>
-
-		<?php if ( isset( $_GET['search'] ) ) : // phpcs:ignore WordPress.Security.NonceVerification.Recommended ?>
-			<h1 class="search-form-title" >
-				<?php // translators: search keyword. ?>
-				<?php printf( esc_html__( 'Showing search results for: %s', 'woocommerce' ), '<strong>' . esc_html( sanitize_text_field( wp_unslash( $_GET['search'] ) ) ) . '</strong>' ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended ?>
-			</h1>
-		<?php endif; ?>
-
-		<form class="search-form" method="GET">
-			<button type="submit">
-				<span class="dashicons dashicons-search"></span>
-			</button>
+	<div class="marketplace-header">
+		<h1 class="marketplace-header__title"><?php esc_html_e( 'WooCommerce Marketplace', 'woocommerce' ); ?></h1>
+		<p class="marketplace-header__description"><?php esc_html_e( 'Grow your business with hundreds of free and paid WooCommerce extensions.', 'woocommerce' ); ?></p>
+		<form class="marketplace-header__search-form" method="GET">
 			<input
 				type="text"
 				name="search"
 				value="<?php echo esc_attr( isset( $_GET['search'] ) ? sanitize_text_field( wp_unslash( $_GET['search'] ) ) : '' ); // phpcs:ignore WordPress.Security.NonceVerification.Recommended ?>"
-				placeholder="<?php esc_attr_e( 'Enter a search term and press enter', 'woocommerce' ); ?>">
+				placeholder="<?php esc_attr_e( 'Search for extensions', 'woocommerce' ); ?>"
+			/>
+			<button type="submit">
+				<span class="dashicons dashicons-search"></span>
+			</button>
 			<input type="hidden" name="page" value="wc-addons">
 			<input type="hidden" name="section" value="_all">
 		</form>
-		<?php if ( '_featured' === $current_section ) : ?>
-			<div class="addons-featured">
-				<?php
+	</div>
+
+	<div class="top-bar">
+		<div id="marketplace-current-section-dropdown" class="current-section-dropdown">
+			<ul>
+				<?php foreach ( $sections as $section ) : ?>
+					<?php
+					if ( $current_section === $section->slug && '_featured' !== $section->slug ) {
+						$current_section_name = $section->label;
+					}
+					?>
+					<li>
+						<a
+							class="<?php echo $current_section === $section->slug ? 'current' : ''; ?>"
+							href="<?php echo esc_url( admin_url( 'admin.php?page=wc-addons&section=' . esc_attr( $section->slug ) ) ); ?>">
+							<?php echo esc_html( $section->label ); ?>
+						</a>
+					</li>
+				<?php endforeach; ?>
+			</ul>
+			<div id="marketplace-current-section-name" class="current-section-name"><?php echo esc_html( $current_section_name ); ?></div>
+		</div>
+		</div>
+
+	<div class="wp-header-end"></div>
+
+	<div class="wrap">
+		<div class="marketplace-content-wrapper">
+			<?php if ( isset( $_GET['search'] ) ) : // phpcs:ignore WordPress.Security.NonceVerification.Recommended ?>
```

Comment:
we should pass the `$search` param from the calling code instead, and use it here. that way we won't have to do phpcs:ignore three times in this template.

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/30391/commits/af9e2b17774d9d8f6a532d54c2e62cdf2bba466a.